ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -2385,10 +2385,12 @@ def verify_provider_storagecluster(sc_data):
     Args:
         sc_data (dict): storagecluster data dictionary
     """
-    log.info(
-        f"allowRemoteStorageConsumers: {sc_data['spec']['allowRemoteStorageConsumers']}"
-    )
-    assert sc_data["spec"]["allowRemoteStorageConsumers"]
+    if version.get_semantic_ocs_version_from_config() < version.VERSION_4_19:
+        log.info(
+            f"allowRemoteStorageConsumers: {sc_data['spec']['allowRemoteStorageConsumers']}"
```

Comment:
same like mentioned above

Reply:
same, not sure if I should change this, since it was designed like this and it is out of the PRs scope